src/rgw/rgw_sal_rados.cc

```diff
@@ -3267,6 +3267,21 @@ int RadosLuaManager::list_packages(const DoutPrefixProvider *dpp, optional_yield
 
   return 0;
 }
+#else
```

Comment:
can we just remove the `#ifdef WITH_RADOSGW_LUA_PACKAGES` around the functions above? they don't actually depend on luarocks, only rgw_lua.cc does